frontend/src/routes/(authenticated)/org/[org_id]/+page.ts

```diff
@@ -158,6 +159,38 @@ export async function _addOrgMember(orgId: UUID, emailOrUsername: string, role:
   return result;
 }
 
+export async function _bulkAddOrgMembers(orgId: UUID, usernames: string[], role: OrgRole): $OpResult<BulkAddOrgMembersMutation> {
+  //language=GraphQL
+  const result = await getClient()
+    .mutation(
+      graphql(`
+        mutation BulkAddOrgMembers($input: BulkAddOrgMembersInput!) {
+          bulkAddOrgMembers(input: $input) {
```

Comment:
we want this mutation to invalidate the gql cache so the list refreshes. I think this means we need to add an invalidation config for this mutation in `gql-client.ts`

Reply:
Attempted in commits c76bebaa and 2e9f433e, but testing shows it's not working yet. Will continue tomorrow.

Reply:
Fixed in commit bbd179d5.